src/Cake.Common.Tests/Fixtures/Tools/DotNetCore/DotNetCoreFixture.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Cake.Core.IO;
+using Cake.Core.Tooling;
+using Cake.Testing.Fixtures;
+
+namespace Cake.Common.Tests.Fixtures.Tools.DotNetCore
+{
+    internal abstract class DotNetCoreFixture<TSettings> : DotNetFixture<TSettings, ToolFixtureResult>
+        where TSettings : ToolSettings, new()
+    {
+        protected override ToolFixtureResult CreateResult(FilePath path, ProcessSettings process)
+        {
+            return new ToolFixtureResult(path, process);
+        }
+    }
+
+    internal abstract class DotNetFixture<TSettings, TFixtureResult> : ToolFixture<TSettings, TFixtureResult>
```

Comment:
Class should be called ``DotNetCoreFixture`2`` to follow naming conventions in Cake (two classes with same name that differ only by generic arguments).